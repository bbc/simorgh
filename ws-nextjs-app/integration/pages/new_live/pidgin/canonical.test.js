/**
 * @service pidgin
 * @pathname /pidgin/new_live/c7p765ynk9qt
 */

import runHeaderTests from '../headerTests';

describe('Canonical', () => {
  describe(pageType, () => {
    console.log('PAGETYPE', pageType);
    runHeaderTests();
  });
});
