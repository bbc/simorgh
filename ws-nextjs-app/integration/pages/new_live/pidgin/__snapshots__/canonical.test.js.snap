// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical New_live I can see the branding 1`] = `".css-1jefi8v{box-sizing:content-box;color:#FFFFFF;fill:currentColor;height:1.25rem;}@media (min-width: 25rem){.css-1jefi8v{height:1.5rem;}}@media (min-width: 37.5rem){.css-1jefi8v{height:1.875rem;}}@media screen and (-ms-high-contrast: active),print{.css-1jefi8v{fill:windowText;}}.css-m04vo2{-webkit-clip-path:inset(100%);clip-path:inset(100%);clip:rect(1px, 1px, 1px, 1px);height:1px;overflow:hidden;position:absolute;width:1px;margin:0;}BBC News, Pidgin"`;
